torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -271,6 +274,33 @@ Operation createUnaryOp(
   };
 }
 
+void MKLDNNLayerNormOp(Stack* stack) {
+  c10::impl::ExcludeDispatchKeyGuard edkg(c10::autograd_dispatch_keyset);
+
+  // enable_cudnn not used
+  pop(stack);
+  auto eps = pop(stack).toDouble();
+
+  Tensor bias{};
+  Tensor weight{};
+  auto bias_ival = pop(stack);
+  TORCH_INTERNAL_ASSERT(bias_ival.isTensor());
```

Comment:
this assertion is already checked in `toTensor` fwiw. although a worse error msg i guess